app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actor_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
```

Comment:
Are you sure the data will be extracted in the correct order?
I consider, that `SELECT id, first_name, last_name` will be better

Reply:
Done